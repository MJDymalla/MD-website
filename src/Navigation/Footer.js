import React, {Component} from 'react';

class Footer extends Component {
    render() {
        return (
            <footer className="footer page-footer">
                <div className="container">
                    <div className="row center">
                        <p>&copy; KIMJ Development</p>
                    </div>
                </div>
            </footer>
        )
    }
}
export default Footer;