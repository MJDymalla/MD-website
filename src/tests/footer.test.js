import React from 'react';
import ReactDOM from 'react-dom';
import ShallowRenderer from 'react-shallow-renderer'; 
import Footer from '../Navigation/Footer';

test('should render header correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<Footer />);
  expect(renderer.getRenderOutput()).toMatchSnapshot();
})



