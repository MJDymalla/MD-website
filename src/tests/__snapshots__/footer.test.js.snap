// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render header correctly 1`] = `
<footer
  className="footer page-footer"
>
  <div
    className="container"
  >
    <div
      className="row center"
    >
      <p>
        Â© KIMJ Development
      </p>
    </div>
  </div>
</footer>
`;
